scripts/001-prepare.sh

```diff
@@ -20,25 +20,24 @@ PROJECT_REPOSITORY=https://github.com/openstack/kolla
 PROJECT_REPOSITORY_PATH=kolla
 SOURCE_DOCKER_TAG=build-$BUILD_ID
 
-# NOTE: For builds for a specific release, the OpenStack version is taken from the release repository.
-if [[ $VERSION != "latest" ]]; then
-    filename=$(curl -L https://raw.githubusercontent.com/osism/release/main/$VERSION/openstack.yml)
-    OPENSTACK_VERSION=$(curl -L https://raw.githubusercontent.com/osism/release/main/$VERSION/$filename | grep "openstack_version:" | awk -F': ' '{ print $2 }')
-fi
-
 . defaults/all.sh
 . defaults/$OPENSTACK_VERSION.sh
 
 export VERSION
 export OPENSTACK_VERSION
 
-git submodule update --remote
+git clone https://github.com/osism/release
```

Comment:
Can we make this optional, to be executed only if the target directory does not exist yet? In the Zuul CI, the repo would already be present via some `required-projects` dependency.

We also then need either the repo path to be configurable or possibly create a symlink. But IMO making on-the-fly changes within the working repo should be done as seldom as possible, so being able to point to a path outside would be my preferred approach.

Reply:
1. part: done
2. part: I'll do that in a follow-up. The kolla repository is currently also being cloned directly. We can then adapt both.